src/test/java/edu/kpi/testcourse/property_tests/AddDeleteTests.java

```diff
@@ -0,0 +1,51 @@
+package edu.kpi.testcourse.property_tests;
+
+import edu.kpi.testcourse.dataservice.DataService;
+import edu.kpi.testcourse.dataservice.UrlAlias;
+import edu.kpi.testcourse.dataservice.User;
+import edu.kpi.testcourse.urlservice.UrlService;
+import io.micronaut.context.BeanContext;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.quicktheories.QuickTheory.qt;
+import static org.quicktheories.generators.SourceDSL.strings;
+
+public class AddDeleteTests {
+
+  static DataService dataService = BeanContext.run().getBean(DataService.class);
```

Comment:
Why don't you use MicronautTest annotation and DI pattern for this test like [here](https://github.com/OlehKyba/url-shrtnr-teacup/blob/main/src/test/java/edu/kpi/testcourse/urlservice/ShortenUrlTests.java)?